src/App.jsx

```
@@ -456,9 +484,10 @@ function App() {
                     <span className="message-username">{message.username}</span>
                     <span className="message-time">{formatTime(message.created_at)}</span>
                   </div>
-                  <div className="message-content">
-                    {message.message}
-                  </div>
+                  <div
+                    className="message-content"
+                    dangerouslySetInnerHTML={{ __html: renderMarkdown(message.message) }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)